src/types/Event.ts

```diff
@@ -17,9 +17,9 @@ const isEvent = (value: unknown): value is Event => {
   return value['@context'] === '/contexts/event';
 };
 
-const isEvents = (value: unknown[]): value is Event[] => {
-  return value?.[0]?.['@context'] === '/contexts/event';
+const areEvents = (value: unknown[]): value is Event[] => {
+  return value.every(isEvent);
```

Comment:
I don't think it matters a lot in this situation, but some might be quicker, when 1 is not an event fail, now you need to always check them all.